skbeam/core/accumulators/projector.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python
+
+from __future__ import division
+import numpy as np
+
+"""
+authors: Mikhail Dubrovin, TJ Lane, Christopher O'Grady
+"""
+
+
+class Projector(object):
+
+    def __init__(self, bin_values, nbins, weights=None, norm=True):
```

Comment:
@tacaswell Hi Tom, I thought about this comment more in light of what you said during the mtg on Monday and I believe you are correct:  the user needs a way to control the binning more easily than with just the "weights" array.  I have added that in the most recent version I just uploaded.